source/_posts/2018-01-31-infrastrucutre-as-code-a-beginners-perspective.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Infrastructure as Code: A beginner's perspective"
+category: infrastructure
+comments: true
+author:
+  name: Federico Kauffman
+  email: federico@wyeworks.com
+  twitter_handle: fedekauffman
+  github_handle: fedekau
+  image: /images/team/federico-kauffman.jpg
+  description: Software Engineer at WyeWorks. Currently working with Javascript and Ruby. Learnaholic.
+---
+
+A couple of months ago I came to realize that DevOps might be something that I like. I was not sure what to do, so I talked to the VP of Engineering at WyeWorks. After that, we found a way for me to experiment and work in a DevOps position for several months.
+
+I left the project I was working on at that moment and I was given a month to learn some basic stuff to be somewhat useful in a DevOps job. That was exactly what I did, since January 2nd I have been reading and doing DevOps stuff 5 days a week 7 hours a day.
+
+In this post I will talk about my perspective about IAC (Infrastructure as Code), what problems it solves, how you can do it, and I will share some resources for you to learn about it. I will also briefly present Terrafrom, as an IAC tool.
+
+<!--more-->
```

Comment:
Remember to remove this 😄

Reply:
Actually this is metadata 🤓 that tells the parse what part is the preview and where the "Read more" link should be placed